### PR TITLE
(DOCSP-30684): Add Bluehawk and Find Unused Files GH actions

### PR DESCRIPTION
**Pull request by cbullinger**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-java/blob/master/REVIEWING.md)

Add the following GH actions to run on PRs:
- Bluehawk - checks for Bluehawk errors (e.g. bad syntax)
- Find Unused Files - doc doctor check for unused files

JIRA - https://jira.mongodb.org/browse/DOCSP-30684
Staging - https://docs-mongodbcom-staging.corp.mongodb.com/kotlin/docsworker-xlarge/docsp-30684-gh-actions/

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [ ] Are all the links working?

### CODE REVIEW

File: source/index.txt [hunk 1]
```diff
@@ -134,3 +134,24 @@ To's and Articles page <https://www.mongodb.com/developer/learn/?content=Article
 
 To ask questions and engage in discussions with fellow developers using
 the Kotlin Driver, see the `forums page <https://www.mongodb.com/community/forums/tag/kotlin>`__.
+
+.. TODO: Add back when University content is available for Kotlin driver
+.. need to re-add /includes/figures/M220J_hero.jpg
+
+.. MongoDB University
```
Comment (rustagir) [inline source/index.txt:141]: Since this is commented out, change lines 129-130 to omit MDB university reference

File: source/index.txt [hunk 2]
```diff
@@ -134,3 +134,24 @@ To's and Articles page <https://www.mongodb.com/developer/learn/?content=Article
 
 To ask questions and engage in discussions with fellow developers using
 the Kotlin Driver, see the `forums page <https://www.mongodb.com/community/forums/tag/kotlin>`__.
```
Comment (rustagir) [inline source/index.txt:136]:
```suggestion
the Kotlin Driver, visit the `MongoDB Developer Community <https://www.mongodb.com/community/forums/tag/kotlin>`__.
```

**Review by rustagir — APPROVED** [review]
small fixes

File: source/index.txt (hunk 2 above)
Comment (cbullinger) [reply source/index.txt:136]: Updated

**Pull request merged**